index.html

```diff
@@ -589,6 +589,12 @@ <h1>Terminology</h1>
         system. Ideally such data is stored in a separate memory inaccessible to the application
         and is only used via abstract operations, such as signing, that do not reveal the secret
         information even to the application using it.</dd>
+      <dt>
+        <dfn>Producer</dfn>
+      </dt>
+      <dd>An entity that can create WoT Thing Descriptions
+        for a specific Thing.</dd>
```

Comment:
Is this a limitation?

Reply:
This exactly corresponds to the term "Generator" that I proposed in my comment.  I'm ok with "Producer" under this specific definition, however.  As long as we are clear that "Exposer != Provider != Producer", since these might all be separate entities.